backstop/tests/table.html

```diff
@@ -1722,35 +1390,14 @@ <h2>Default</h2>
             </div>
             <div class="iui-cell iui-main-column">
               <div class="iui-cell-start-icon">
-                <!-- Use filetype-pdf.svg from @itwin/itwinui-icons-color package. -->
-                <svg
-                  viewBox="0 0 16 16"
-                  aria-hidden="true"
-                >
-                  <path
-                    d="m11 3v-3h-9v16h12v-13z"
-                    fill="#d16c00"
-                  />
-                  <path
-                    d="m11.9665 9.6309a3.95941 3.95941 0 0 0 -1.3446-2.6309h1.3781v-1h-2.5921a1.4911 1.4911 0 0 0 -2.8158 0h-2.5921v1h1.3781a3.95941 3.95941 0 0 0 -1.3446 2.6309.99.99 0 0 0 -.5335.8691 1 1 0 0 0 2 0 .98412.98412 0 0 0 -.4709-.8268 2.98911 2.98911 0 0 1 1.7037-2.3788 1.49069 1.49069 0 0 0 2.5344 0 2.98911 2.98911 0 0 1 1.7037 2.3788.98412.98412 0 0 0 -.4709.8268 1 1 0 0 0 2 0 .99.99 0 0 0 -.5335-.8691zm-3.9665-2.6309a.5.5 0 1 1 .5-.5.5.5 0 0 1 -.5.5z"
-                    fill="#fff"
-                  />
-                  <path
-                    d="m14 3h-3v-3z"
-                    fill="#924b00"
-                  />
-                </svg>
+                <svg-filetype-vector aria-hidden="true"></svg-filetype-vector>
```

Comment:
and comment said its pdf...

Reply:
then comment was wrong 
![image](https://user-images.githubusercontent.com/9084735/131369333-bdf1782d-c361-4934-93ed-76acb9e46983.png)